lib/swaggerUtil.js

```diff
@@ -106,7 +106,8 @@ function validateObject(specs, obj, disc) {
     else if (!specs.definitions[obj[disc]]) {
         validationErrors.push({
             message: 'model definition not found for ' + disc +
-                ': ALL model definitions must be defined in root swagger document'
+                ': ALL model definitions must be defined in root swagger document. ' +
+                'This can be accomplished by turning ref compilation "on"'
```

Comment:
I think this message still needs to be updated, right?
I'm OK with authoring the docs on the wiki first so that we can get the link for the message.